TACC/protx-dashboard
# Task/COOKS-339: update state wide plot

## Overview: ##

Update state-wide plot to match Juliana's.

I also fixed the colors of the chart so that they match the map (One of the colors was wrong).

## Related Jira tickets: ##

* [COOKS-339](https://jira.tacc.utexas.edu/browse/COOKS-339)

## Summary of Changes: ##

## Testing Steps: ##
1. Go to analytics page
2. Confirm new chart
3. Confirm that 3 colors on chart match 3 colors on map (i.e. low, medium, high)

## UI Photos:
![Screen Shot 2022-10-24 at 3 40 08 PM](https://user-images.githubusercontent.com/8287580/197628929-b52cf4ec-3c0e-42a4-a246-5fa7038e2a5f.png)
